lametro/forms.py

```diff
@@ -68,7 +68,7 @@ def _topic_search(self, sqs):
 
         topic_filter = SQ()
         for term in terms:
-            topic_filter &= SQ(topics__in=[term])
+            topic_filter &= SQ(topics__in=[term]) | SQ(bill_type__in=[term])
```

Comment:
> I think it would be a good idea to match users' tag searches over all of those values.

This is a great idea. As a stopgap to the combined index field, could we update this query to search all of the fields you mention?